ChainEval/configs/gsm8k/gemma2b-it/cot_fewshot.yaml

```diff
@@ -15,7 +15,7 @@ dataset_config:
     cache_dir: _cache_data/
 
 prompt_config:
-  prompt_name: GSM8K_COT_FEWSHOT
+  prompt_name: GSM8K_COT_FEWSHOT_V_DIVERSE
```

Comment:
@Anshumaan-Chauhan02 cot_fewhsot.yaml should have `prompt_name: GSM8K_COT_FEWSHOT`